jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/bsm/BsmCoder.java

```diff
@@ -63,9 +66,11 @@ public void decodeFromHexAndPublish(InputStream is, String topic)
                publish(topic, decoded.toJson());
          }
          if (empty) {
+        	data.info("Empty file was received");

```

Comment:
Messages logged to application log file and data log file should be consistent (the same if appropriate).